pom.xml

```diff
@@ -71,7 +86,7 @@
 					<version>3.2</version>
 					<configuration>
 						<source>1.8</source>
-						<target>1.7</target>
+						<target>1.8</target>
```

Comment:
Only this one is making me nervous. I think we should stick with Java 7 as a target as long as we don't need Java 8.